airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/migrate_to_manifest_only/declarative_component_schema.py

```diff
@@ -0,0 +1,1596 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+# generated by datamodel-codegen:
+#   filename:  declarative_component_schema.yaml
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import Any, Dict, List, Optional, Union
+
+from pydantic import BaseModel, Extra, Field
+from typing_extensions import Literal
+
+## This is a stupid hack. I've copied the current version of the models from the generated code and pasted them here since we can't access the CDK from the current directory.
```

Comment:
```suggestion
## WARNING
## This is a hack. I've copied the current version of the models from the generated code and pasted them here since we can't access the CDK from the current directory.
## These are from CDK 4.5.3.
```